application/views/user_result.php

```diff
@@ -36,51 +36,47 @@
     <table class="table table-hover">
         <thead>
         <tr>
-            <th>序号</th>
-            <th>姓名</th>
-            <th>性别</th>
-            <th>手机号</th>
-            <th>证件类型</th>
-            <th>证件编号</th>
-            <th>北大赛</th>
-<!--            <th>公路日</th>-->
-<!--            <th>山地日</th>-->
-            <th>住宿</th>
-            <th>5.14晚餐</th>
-            <th>5.15午餐</th>
-            <th>清真</th>
-            <th>费用</th>
+          <th>序号</th>
```

Comment:
Please revert back the indentation change to avoid so many unnecessary changes.
Also for the following part.